PasscodeLock.xcodeproj/project.pbxproj

```diff
@@ -447,16 +447,16 @@
 				TargetAttributes = {
```

Comment:
- Please remove the .swift_version file
- Please add `s.swift_version = '5.0'` to the podspec
- Make sure that the project is building, and fix warnings that are not triggered by swiftlint if there is some (Robin can show you how to recognise them)